wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 {% load static i18n %}
+{% get_current_language as LANGUAGE_CODE %}
 <html class="no-js" lang="{{ LANGUAGE_CODE|default:"en-gb" }}">
```

Comment:
I'm thinking it might be best to ditch the `|default:"en-gb"` from this line. Whether or not a current language has been specified, I think `get_current_language()` would always return the value of `settings.LANGUAGE_CODE` - and I imagine it's very rare for that not to have a value.

Reply:
Good point! Looks like Django will supply a default language code of `en-us` if there isn't one specified in settings, so the default here would never get picked up even if we wanted it to...